src/react/account/AccountEditCommonLayout.tsx

```diff
@@ -52,118 +25,118 @@ export const CommonPolicyLayout = ({
   isReadOnly?: boolean;
   control: Control<{ policyDocument: string }>;
   policyDocument: string;
-  errors?: { policyDocument?: { message?: string } };
+  errors?: {
+    policyName?: { message?: string };
+    policyDocument?: { message?: string };
+  };
   isDirty: boolean;
   isValid: boolean;
-  handleCancel: (e: MouseEvent) => void;
+  handleCancel: (e: ChangeEvent<HTMLInputElement>) => void;
 }) => {
   const isCreateMode = !policyArn;
   return (
-    <FormContainer>
-      <F.Form onSubmit={onSubmit}>
-        <F.Title>
-          Policy {isCreateMode ? 'Creation' : !isReadOnly ? 'Edition' : ''}
-        </F.Title>
-        <FittingHr />
-        {isCreateMode && (
-          <SecondaryText> All * are mandatory fields </SecondaryText>
-        )}
-        <StyledFieldSet>
-          <StyledLabel htmlFor="policyName">
-            Policy Name{isCreateMode && <> *</>}
-          </StyledLabel>
-          {policyNameField}
-        </StyledFieldSet>
-        {policyArn && (
-          <StyledFieldSet>
-            <StyledLabel htmlFor="policyARN">Policy ARN</StyledLabel>
-            <span>{policyArn}</span>
-            <Clipboard text={policyArn} />
-          </StyledFieldSet>
-        )}
-        <F.Fieldset>
-          <F.Label htmlFor="policyDocument">
-            Policy Document{isCreateMode && <> *</>}
-          </F.Label>
-          <Box display="flex" flex-direction="row" gap={1} mt={2}>
-            <Controller
-              control={control}
-              name="policyDocument"
-              rules={{
-                required: 'The policy document is required',
-              }}
-              render={({ field: { onChange, value } }) => (
-                <Editor
-                  language="application/json"
-                  width="33rem"
-                  height="20rem"
-                  onChange={onChange}
-                  value={value}
-                  readOnly={isReadOnly}
-                  beforeMount={(monaco: Monaco) => {
-                    monaco.languages.json.jsonDefaults.setDiagnosticsOptions({
-                      validate: true,
-                      schemas: [
-                        {
-                          uri: 'http://myserver/foo-schema.json', // id of the first schema
-                          fileMatch: ['*'],
-                          schema: policySchema,
-                        },
-                      ],
-                    });
-                  }}
-                />
-              )}
+    <Form
+      onSubmit={onSubmit}
+      layout={{
+        kind: 'page',
+        title: `Policy ${
+          isCreateMode ? 'Creation' : !isReadOnly ? 'Edition' : ''
+        }`,
+      }}
+      requireMode={isCreateMode ? 'partial' : 'all'}
+      rightActions={
+        isReadOnly ? (
+          <Button variant="outline" label="Close" onClick={handleCancel} />
+        ) : (
+          <Stack gap="r16">
+            <Button variant="outline" label="Cancel" onClick={handleCancel} />
```

Comment:
nit: I think without a type="button" it will be identified as a "submit" button by default. 
```suggestion
            <Button variant="outline" label="Cancel" type="button" onClick={handleCancel} />
```